CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contributing
+
+To contribute to this project please carefully read this document.
+
+## Setup
+
+`macos-wakatime` is written in [Swift](https://www.swift.org/).
+
+## Branches
+
+This project currently has two branches
+
+- `main` - Default branch for every new `feature` or `fix`
+- `release` - Branch for production releases and hotfixes
+
+## Testing and Linting
+
+Run `xcodebuild` before creating any pull requests, or your PR won’t pass the automated checks.
```

Comment:
Mention `xcodebuild` is the same as running/building the project with Xcode, and that building the project with Xcode also runs the linter.

Don't want people to get confused thinking they have to use the command line when just Xcode is enough.